src/users/interfaces/serialized-user-metrics.ts

```diff
@@ -28,4 +28,7 @@ export interface SerializedUserMetrics {
     total_hours: number;
     last_checked_in: string | null;
   };
+  pool_points?: {
+    [pool: string]: number;
```

Comment:
Do we want to just explicitly type out the pools here, will be easier to understand if someone has to come back to this in the future. We will never add new pools, right?

Reply:
Yeah sounds good